components/raftstore/src/store/fsm/apply.rs

```diff
@@ -2681,9 +2707,10 @@ where
         self.delegate.apply_state.set_last_commit_index(cur_state.0);
         self.delegate.apply_state.set_commit_index(cur_state.1);
         self.delegate.apply_state.set_commit_term(cur_state.2);
-
+        // Before draining the entire entries, we need to update the entry_mem_size.
```

Comment:
Why?

Reply:
Because I think if we drain the apply's entries, we need to update the entries data size change.

Reply:
Ping~

Reply:
Isn't it included in drop method?

Reply:
Drop not recalculate data size change, only flush the size change.

Reply:
Did you mean we don't care the data mem size change here, we can directly statistics it at new and drop?

Reply:
Excactly.

Reply:
Fixed.